### ISSUE TITLE
Improve ratings display to make it nicer somehow?

### ISSUE THREAD

**Issue by acorncom** [ember-learn/ember-help-wanted#3]
Stars or potentially something else slightly cutesy

**Comment by acorncom** [comment]
Depends on #1

**Comment by MelSumner** [comment]
Can you clarify what you mean by this one?

**Comment by ijlee2** [comment]
I don't think adding support for rating issues will be in my immediate plans for the client and server apps. Let's open a new issue if, at a later point, rating issues can be a feasible problem. (related issue: #1)
